docs/ums/README.md

```diff
@@ -32,24 +31,25 @@ ums.signUp({
       city: "Apeldoorn",
       country: "NL"
     }
-  }).subscribe(
+}).subscribe(
     user => {..do something with registered user}, 
-    error=>{..handle error}
-  );  
+    error=> {..handle error}
+);  
+
 //before < JS SDK v5.0.0.
 ums.signUp({    
     email: "user@company.com",    
-    password: "my_password"},
-    {
-      age: 25, 
-      address: { 
+    password: "my_password"
+}, {
```

Comment:
I think we did change in SDK which supposes to accept only 1 object, instead of 2. With 2 is not comfortable to use